cocos/core/gfx/webgl/webgl-queue.ts

```diff
@@ -9,8 +9,6 @@ export class WebGLGFXQueue extends GFXQueue {
     public numInstances: number = 0;
     public numTris: number = 0;
 
-    private _isAsync: boolean = false;
-
     public initialize (info: IGFXQueueInfo): boolean {
```

Comment:
forceSync haven't been used in `initialize`, doesn't seem right

Reply:
WebGL queues are inheritantly synchronized, no asycn functionalities present for now